semantic/src/site/globals/site.overrides

```diff
@@ -906,8 +906,7 @@ a:hover {
     background-color: @surfacesLight;
     padding: 15px;
     border-radius: @borderRadius;
-    border: 1px solid @surfacesMed;
-    min-width: 400px;
+    border: 1px solid @surfacesMed;    
     h4 {
```

Comment:
Removed this min-width site wide.  Couldn't see it being of any benefit anywhere, and it would stuff up the mobile rendering.